app/desmos/cmd/root.go

```diff
@@ -274,6 +275,7 @@ func newApp(logger log.Logger, db dbm.DB, traceStore io.Writer, appOpts serverty
 		cast.ToUint(appOpts.Get(server.FlagInvCheckPeriod)),
 		app.MakeTestEncodingConfig(), // Ideally, we would reuse the one created by NewRootCmd.
 		appOpts,
+		wasm.DisableAllProposals,
```

Comment:
Shouldn't we get this list from somewhere? Otherwise we will have all proposals always disabled. Or am I missing something?

Reply:
No, currently it is hard coded, and it requires all the nodes having the same accepted list. Hope they can have the on chain parameter in the future.
Here I use the previous setting which disabled all proposals.